### PR TITLE
BaSD-MS-62: modified input for Final-Form and admins Form

### PR DESCRIPTION
**Pull request by leimeterjoaquin**
This PR includes Admins entitys form. Also adds a regex folder with validations

### CODE REVIEW

File: src/Components/Admin/Admins/Form/index.js [hunk 1]
```diff
@@ -35,92 +29,42 @@ const AdminsForm = () => {
...
-      }
+  const validate = (formValues) => {
+    console.log('validating');
+    const errors = {};
+    if (!emailValidation.test(formValues.email)) {
+      errors.email = 'Invalid email';
     }
+    if (!passwordValidation.test(formValues.password)) {
+      errors.password = 'At least 8 characters, one letter and one number';
+      console.log('error');
```
Comment (aquilesmariani) [inline src/Components/Admin/Admins/Form/index.js:65]: Let's get rid of console.log

File: src/Components/Admin/Admins/Form/index.js [hunk 2]
```diff
@@ -147,37 +91,41 @@ const AdminsForm = () => {
...
-          showError={passwordError}
-          text={'Password is too short. It must have at least 8 characters.'}
-        />
-        <Button type="Submit" disabled={canSave} />
-      </form>
+      <Form
+        onSubmit={(formValues) => {
+          setFormValues(formValues);
+          setShowModal(true);
+        }}
```
Comment (aquilesmariani) [inline src/Components/Admin/Admins/Form/index.js:98]: Could you move the arrow function outside?

File: src/Components/Admin/Admins/Form/index.js [hunk 3]
```diff
@@ -147,37 +91,41 @@ const AdminsForm = () => {
...
+        render={(formProps) => (
+          <form className={styles.form} onSubmit={formProps.handleSubmit}>
+            <h2>Form</h2>
+            <Field
+              name="email"
+              label="email"
+              placeholder="example@geemail.com"
+              component={Input}
+              disabled={formProps.submitting}
+              validate={(value) => (value ? undefined : 'please enter your email')}
```
Comment (aquilesmariani) [inline src/Components/Admin/Admins/Form/index.js:110]: What if we create a shared function for validating email and password?

File: src/Components/Admin/Admins/Form/index.js [hunk 4]
```diff
@@ -35,92 +29,42 @@ const AdminsForm = () => {
...
-        setPasswordError(true);
-        setCanSave(true);
-      } else if (!emailValue.includes('@')) {
-        setEmailError(true);
-        setCanSave(true);
-      } else if (!emailError && !passwordError) {
-        setCanSave(false);
-      }
+  const validate = (formValues) => {
+    console.log('validating');
```
Comment (aquilesmariani) [inline src/Components/Admin/Admins/Form/index.js:58]: Another console.log

File: src/Components/Admin/Admins/Form/index.js (hunk 3 above)
Comment (leimeterjoaquin) [reply src/Components/Admin/Admins/Form/index.js:110]: I will create a shared function for each. So they can be used separately on each entity

Comment (leimeterjoaquin) [reply src/Components/Admin/Admins/Form/index.js:110]: Check if my solution was ok. I think it's scalable

**Review by aquilesmariani — APPROVED** [review]

**Review by ignacioaristo — APPROVED** [review]

**Pull request merged**